hw3/style.css

```diff
@@ -0,0 +1,50 @@
+header{
+    background-color: red;
+    color: white;
+    font-weight: bold;
+    font-size: 30px;
+    text-align: center;
+    width: 760px;
+    margin: auto;

```

Comment:
If you are setting "margin: auto" it means margin for top/bottom/left/right - and then you rewrite it with "margin-bottom". Better to fix into one line "margin: 0 auto 20px"

Reply:
done